frontend/rest-frontend/src/main/java/com/alexkoveckiy/frontend/rest/MyRestController.java

```diff
@@ -19,15 +20,29 @@
 @RestController
 @RequestMapping(method = RequestMethod.POST,
         consumes = MediaType.APPLICATION_JSON_VALUE,
-        produces = MediaType.APPLICATION_JSON_VALUE)
+        produces = MediaType.APPLICATION_JSON_VALUE,
+        path = "server")
 public class MyRestController {
 
     @Autowired
     private FirstRouter firstRouter;
 
-    @RequestMapping(path = "server")
-    public Response<?> getRequest(@RequestBody final Request<?> request) {
+    @Autowired
+    private AuthorizationRouter authorizationRouter;
```

Comment:
Because public requests go only to 3 handlers. They should not have ability going to other routers. If I route them to FirstRouter, same as private requests, they could go to some private handler.